### ISSUE TITLE
op-geth init gives "head block is not reachable" error

### ISSUE THREAD

**Issue by SyedMuhamadYasir** [ethereum-optimism/op-geth#420]
I am trying to set up optimism on a local testnet (ethereum PoS)

however, when doing the following step (**Initialize op-geth**)

`build/bin/geth init --datadir=datadir genesis.json`

i get the following error ("head block is not reachable"):

<img width="1201" alt="Screenshot 2024-10-29 at 13 21 24" src="https://github.com/user-attachments/assets/183192de-71df-46cf-8d04-81cb40e1a954">

what can i do to solve this?

**Comment by feynman-x** [comment]
I get this too. Have you find a way to solve ?

**Comment by lei335** [comment]
I got this too, then i did the step 'build/bin/geth init --datadir=datadir genesis.json' again, it worked fine.

**Comment by SyedMuhamadYasir** [comment]
> I get this too. Have you find a way to solve ?

just continue with the setup, the head block error is not important

**Comment by guardianf** [comment]
yeah, i met this problem too. and i just use "ln -s ,/build/bin/geth /bin/geth" to solve  this problem

> I got this too, then i did the step 'build/bin/geth init --datadir=datadir genesis.json' again, it worked fine.

**Comment by 0x4k-helios** [comment]
Hi, @SyedMuhamadYasir 
How to check the geth is fully synced?
I tried on the documentation, but it shows eth.blockNumber is 0 consistently

**Comment by mikelear** [comment]
I'm having this same issue , i get this error and carried on but geth is always at Block 0 and when i connect and propose a block from Beacon i get errors that the parent is nil , which seems linked I'm using the latest builds , has anyoen setup a PoS Testnet recently and isn't seen these issues ?

**Comment by cliff0412** [comment]
I got this too
